src/tests/ftest/control/dmg_network_scan.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from __future__ import print_function
+
+import os
+import re
+import socket
+
+from avocado.utils import process
+from control_test_base import ControlTestBase
+
+
+class NetDevList(object):
```

Comment:
If you where to instead include the `__eq__` method in the `NetDev` class definition, you should not need this class

Reply:
But given I do that I would then only be able to compare single NetDev objects against each other. I added this class so I could compare the whole list of NetDev objects. Or am I wrong on that?